modules/ROOT/pages/troubleshooting-visualization.adoc

```diff
@@ -0,0 +1,86 @@
+
+In the *Metrics* section, you can select one of the following metrics:
+
+* *Avg response time*
+* *Avg throughput*
+* *Failure rate*
+* *Avg CPU utilization*
+* *Avg memory utilization*
+
+For the selected time range, _failure rate_ is equal to the total number of failures divided by the total number of requests.
+
+For average response time, average throughput, and failure rate, you can select *Inbound* or *Outbound* metrics.
+If you select inbound, the value that appears on the application is the weighted average of that particular metric for all inbound connections to that node.
+If you select outbound, the value is the weighted average of that metric for all outbound connections.
+
+In the *Time Range* section, you can select the time window used to calculate the average values of the selected metric.
+
+=== Metrics Notation
+
+To avoid long numbers, Anypoint Visualizer uses unit prefixes to simplify your view. For example:
+
+|===
+| Avg throughput | 66.7m TPM | 66.7 milli transactions per minute | 0.0667 TPM
+|===
+https://en.wikipedia.org/wiki/Metric_prefix[Here is a quick review of metric prefixes].
+
+
+
+== View Metrics for One Service in your Network
+
+When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
+Anypoint Visualizer displays information about the node in a card. This card is divided into the following tabs:
+
+The *Details* tab contains the *Details* section, which provides information about the node.
+
+The *Monitoring* tab contains:
+
+Metrics summary::
+Provides monitoring metrics for the selected node, including average response time, average throughput, failure rate, average CPU utilization, and average memory utilization. This shows either inbound or outbound values for average response time, average throughput, and failure rate, depending on what you’ve selected in the *Metric* card.
+
+Metrics chart:: Shows the time-series chart of the selected metric in the *Metric* card.
+
+Within the *Monitoring* tab, you can also select *View logs* or *View dashboard* to navigate to the associated logs or dashboard in Anypoint Monitoring.
+
+== View Metrics for a Connection in Your Network
+
+If the selected metric is compatible, the canvas also displays metrics on the connections. When you select a connection within the graph, Anypoint Visualizer displays the edge’s metrics details in a card.
+For more information about which connections support this feature, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].
+
+[NOTE]
+Edges displayed between services are active, meaning that they appear only if there has been traffic in the past seven days. Nonactive edges do not appear on the canvas.
+== Filtering your View
```

Comment:
This did not appear as a heading in the docs

Reply:
It showed up in the note

Reply:
```suggestion

== Filtering your View
```